Assets/MRTK/Tests/PlayModeTests/PointerBehaviorTests.cs

```diff
@@ -227,10 +227,39 @@ public IEnumerator TestGrab()
         /// Tests that the teleport pointer functions as expected
         /// </summary>
         [UnityTest]
-        public IEnumerator TestTeleport()
+        public IEnumerator TestTeleportAndFloorHeight()
         {
             var iss = PlayModeTestUtilities.GetInputSimulationService();
+            ExperienceScale originalExperienceScale;
+            float originalProfileFloorHeight;
 
+            float floorHeight = 1.3f;
+
+
+            // MRTK has already been created by SetUp prior to calling this,
+            // we have to shut it down to re-init with the custom input profile which
+            // has our floorHeight value set
```

Comment:
Would it be possible to switch profiles like the profile-switching tests do?

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/6e061451d7caed1fcb7c324baf92be293efda4cf/Assets/MRTK/Tests/PlayModeTests/ChangeActiveProfileTests.cs#L36-L41

Reply:
Since some of the scene content gets initialized on awake, just switching the active profile instance doesn't reinitialize the scene in the way we want.